lib/wor/paginate/paginate.rb

```diff
@@ -42,6 +46,10 @@ def param_limit
       params[Config.per_page_param].to_i unless params[Config.per_page_param].nil?
     end
 
+    def includes?(options)
+      !options[:include].nil?
```

Comment:
```suggestion
      options[:include].present?
```

Reply:
this would be easier to read

Reply:
I agree

Reply:
Isn't `present?` a Rails method? I thought this gem was intended to be plain ruby so even though this check could be written differently `present?` is not a real solution

Reply:
Didnt realize this was in plain ruby....nvm then

Reply:
Oops!